README.md

```diff
@@ -12,7 +12,10 @@ kitchen converge
 4. delivery
 5. build-node
 
-#### Must have a license key to build Delivery
+**The intial `kitchen converge` will fail because of a missing *delivery.license* file. You can obtain a *delivery.license* file from Chef Internal Slack channel #delivery-support**
```

Comment:
What about pointing people to https://www.chef.io/delivery/ to get a delivery key and removing the internal stuff?